L/libjulia/common.jl

```diff
@@ -246,7 +261,7 @@ function build_julia(version)
         push!(dependencies, Dependency(PackageSpec(name="LibGit2_jll", version=v"0.28.2")))
     elseif version.major == 1 && version.minor == 6
         push!(dependencies, Dependency(PackageSpec(name="OpenBLAS_jll", version=v"0.3.10")))
-        push!(dependencies, Dependency(PackageSpec(name="libLLVM_jll", version=v"9.0.1")))
+        push!(dependencies, Dependency(PackageSpec(name="LLVM_full_jll", version=v"11.0.0")))
```

Comment:
I guess we need a `libLLVM_jll` for LLVM 11...

Reply:
There now is a libLLVM_jll for 11